account_move_transactionid_import/tests/test_completion_transaction_id.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from datetime import datetime
+from odoo.modules import get_resource_path
+from odoo.tests import SingleTransactionCase
+from odoo.tools import convert_file
+
+
+class TestCompletionTransactionId(SingleTransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        convert_file(
+            cls.cr,
+            'account',
+            get_resource_path('account', 'test', 'account_minimal_test.xml'),
```

Comment:
I don't know is this needed anymore as in v12 test env for accounting is different

Reply:
I don't know on Travis but running locally with pytest it was needed.